features/step_definitions/web_steps.rb

```diff
@@ -53,6 +53,11 @@ def with_scope(locator)
   click_button(button)
 end
 
+#Alternative method to click_button(button), used when button can not be found
+When /^(?:|I )find the "([^"]*)" button and click it$/ do |button|
```

Comment:
Actually, let's reword this slightly for readability:
`When /^(?:|I )find and click the "([^"]*)" button$/`